src/GeneralStateTestsFiller/stPreCompiledContracts2/CALLBlake2fFiller.json

```diff
@@ -0,0 +1,418 @@
+{
+  "CALLBlake2f": {
+    "env": {
+      "currentCoinbase": "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+      "currentDifficulty": "0x20000",
+      "currentGasLimit": "10000000000",
+      "currentNumber": "1",
+      "currentTimestamp": "1000",
+      "previousHash": "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+    },
+    "expect": [
+      {
+        "indexes": {
+          "data": [0],
+          "gas": -1,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0x62305ad4d48dade8269ef60a1bcb8b7bef6e479e643b5ac1f8017e6422ce89fb",
+              "0x03": "0x62d09ecaa81d095e855540dcbc07bd0feb3d4f5e5e50541260ed930f027cfd8d"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999999915711"
```

Comment:
if possible do not use balance on post condition. in case some gas calculation rule change, this will affect this test.

Reply:
If we don't judge balance how do we test gas calculation?

Reply:
if test specificly need to test gas calculation it is ok to use balance in post section. 
there is one more way to test balance 
```
[[0]] (BALANCE 0x...) 
```

Reply:
Gas calculation varies per call, so we need the calculations.  Can all remain or should I only leave in a representative few?